content/en/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -7,101 +7,81 @@ min-kubernetes-server-version: 1.22
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
 
-Windows HostProcess containers enable you to run containerized 
-workloads on a Windows host. These containers operate as 
-normal processes but have access to the host network namespace, 
-storage, and devices when given the appropriate user privileges. 
+Windows HostProcess containers enable you to run containerized
+workloads on a Windows host. These containers operate as
+normal processes but have access to the host network namespace,
+storage, and devices when given the appropriate user privileges.
 HostProcess containers can be used to deploy network plugins,
-storage configurations, device plugins, kube-proxy, and other 
-components to Windows nodes without the need for dedicated proxies or 
+storage configurations, device plugins, kube-proxy, and other
+components to Windows nodes without the need for dedicated proxies or
 the direct installation of host services.
 
-Administrative tasks such as installation of security patches, event 
-log collection, and more can be performed without requiring cluster operators to 
-log onto each Window node. HostProcess containers can run as any user that is 
-available on the host or is in the domain of the host machine, allowing administrators 
-to restrict resource access through user permissions. While neither filesystem or process 
-isolation are supported, a new volume is created on the host upon starting the container 
-to give it a clean and consolidated workspace. HostProcess containers can also be built on 
-top of existing Windows base images and do not inherit the same 
-[compatibility requirements](https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility) 
-as Windows server containers, meaning that the version of the base images does not need 
-to match that of the host. HostProcess containers also support 
+Administrative tasks such as installation of security patches, event
+log collection, and more can be performed without requiring cluster operators to
+log onto each Window node. HostProcess containers can run as any user that is
+available on the host or is in the domain of the host machine, allowing administrators
+to restrict resource access through user permissions. While neither filesystem or process
+isolation are supported, a new volume is created on the host upon starting the container
+to give it a clean and consolidated workspace. HostProcess containers can also be built on
+top of existing Windows base images and do not inherit the same
+[compatibility requirements](https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility)
+as Windows server containers, meaning that the version of the base images does not need
+to match that of the host. HostProcess containers also support
```

Comment:
@marosset  Should we mention that it would still be recommended since the image is unpacked on the host and could lead to additional storage?

Reply:
Oh yea, we should mention this - especially since windows images can be very large.

Reply:
Updated to recommend that you use the same base image as your WS container workload